WordPress/src/main/res/values/styles_toolbar.xml

```diff
@@ -47,6 +47,7 @@
         <item name="android:textColor">?attr/colorOnSurface</item>
         <item name="fontFamily">serif</item>
         <item name="android:textStyle">bold</item>
+        <item name="android:textSize">@dimen/text_sz_extra_large</item>
```

Comment:
This change decreases the size of the title in an expanded state which is what we need in the My Site tab with no sites but this style is used in the Notifications and Reader tab. So the size of the Notifications/Reader title becomes smaller than what we had originally in the app before this PR. 

| Title UI after this change  |   Expected |
|---|---|
| ![Screenshot_20221102-133559](https://user-images.githubusercontent.com/17463767/199435474-08418bb0-a867-47b0-871b-777ff3d24e98.png) | ![Screenshot_20221102-133606](https://user-images.githubusercontent.com/17463767/199435514-a9c80931-ce19-47a7-a910-eecb63f08330.png)|